cairo/tests/ethereum/crypto/test_alt_bn128.py

```diff
@@ -222,6 +222,16 @@ def test_bnp12_point_at_infinity(self, cairo_run):
         def test_bnp12_eq(self, cairo_run, a: BNP12, b: BNP12):
             assert cairo_run("BNP12__eq__", a, b) == (a == b)
 
+        @given(p=...)
+        def test_bnp12_double(self, cairo_run, p: BNP12):
+            try:
+                expected = p.double()
+            except OverflowError:  # fails for large points
+                with pytest.raises(Exception):  # Hint error
+                    cairo_run("bnp12_double", p)
```

Comment:
if we expect a hint error, can we ensure that the error string contains this error ?